src/Game.js

```diff
@@ -76,11 +80,14 @@ export default class Game {
     this.puck.velocity = { x: 0, y: 0 };
     this.puck.acceleration = { x: 0, y: 0 };
     this.puck.move(PUCK_POSITION.X, PUCK_POSITION.Y);
+    this.puck.resetTrace();
   }
 
   render() {
     this.clear();
     Object.values(this.groups).forEach((group) => group.render(this.ctx));
+    document.querySelector('.js-tries').innerHTML = this.tries;
+    document.querySelector('.js-charges').innerHTML = this.chargesCounter;
```

Comment:
Is there a point in updating this 60 times per second? I think this text should be updated only when it's changed. DOM manipulations are expensive.